### PR TITLE
Update gridfinity-rebuilt.scad

### PR DESCRIPTION
**Pull request by rhandel**
Added clarity to customizer.

### CODE REVIEW

File: gridfinity-rebuilt.scad [hunk 1]
```diff
@@ -20,36 +20,53 @@ https://github.com/kennetek/gridfinity-rebuilt-openscad
...
 
-color("tomato")
-gridfinityEqual(n_divx = 2, n_divy = 1, style_tab = 1, enable_scoop = true);
+/*[Scoop]*/
+scoop = true; //[true:false]
 
+// ===== Commands ===== //
 
+color("aqua")
+ gridfinityEqual(n_divx = divx, n_divy = divy, style_tab = tab_style, enable_scoop = scoop);
```
Comment (Ruudjhuu) [inline gridfinity-rebuilt.scad:66]: Also make this change to line 106. Keep the examples in sync.

File: gridfinity-rebuilt.scad [hunk 2]
```diff
@@ -20,36 +20,53 @@ https://github.com/kennetek/gridfinity-rebuilt-openscad
...
-enable_lip = true; // toggle the lip on the top of the bin that allows stacking
+/* [Divisions] */
+// DivX Number of x Divisions
+divx = 1; //[1,2,3,4,5,6,7,8,9,10]
+
+// DivY Number of y Divisions
+divy = 1; //[1,2,3,4,5,6,7,8,9,10]
+/* [Holes] */
+// Bottom screw/magnate holes?
+enable_holes = true; // [ture:false]
```
Comment (Ruudjhuu) [inline gridfinity-rebuilt.scad:42]:
```suggestion
enable_holes = true; // [true:false]

```

**Review by Ruudjhuu — CHANGES_REQUESTED** [review]

**Comment by Ruudjhuu** [comment]
Great work!

I think you should also update the "Global Parameters" section of the README file.

File: gridfinity-rebuilt.scad (hunk 2 above)
Comment (rhandel) [reply gridfinity-rebuilt.scad:42]: Accepted  Will change to true

File: gridfinity-rebuilt.scad (hunk 1 above)
Comment (rhandel) [reply gridfinity-rebuilt.scad:66]: 106 is a manual example.

I am not sure it needs to reflect the customizer values.  Those are left to more experienced users to view.  Ask the original author.  I was intending on only neatening up the customizer with these changes.  Making it easier for novice/newbies to use.  I am also contemplating some other more advanced changes to the "Other Features" category.  I am big on making code fun for novices and better documented for hackers.  ;-)

This is great OpenSCAD code to learn from.

Thanks Kenneth/All.

**Comment by rhandel** [comment]
Thanks.

I had to look up the formatting myself, since I infrequently use Open SCAD.  I love OpenSCAD but find it hard to do some types of transforms.  I really want to learn more myself.

RAH

**Pull request merged**